README.md

```diff
@@ -46,23 +46,23 @@ The following chart shows the time it takes to run the 22 TPC-H queries against
 using a single executor with 8 cores. See the [Comet Benchmarking Guide](https://datafusion.apache.org/comet/contributor-guide/benchmarking.html)
 for details of the environment used for these benchmarks.
 
-When using Comet, the overall run time is reduced from 626 seconds to 290 seconds, a 2.2x speedup.
+When using Comet, the overall run time is reduced from 616 seconds to 275 seconds, a 2.2x speedup.
```

Comment:
```suggestion
When using Comet, the overall run time is reduced from 616 seconds to 275 seconds, a 2.25x speedup.
```

Reply:
I'd prefer to stick with single-digit precision for this, since the results vary slightly with each run